src/openforms/js/components/admin/form_design/registrations/zgw/fields/CatalogueSelect.js

```diff
@@ -1,45 +1,15 @@
 import {useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
 import {FormattedMessage} from 'react-intl';
-import {useAsync} from 'react-use';
 
-import {
-  CatalogueSelect as GenericCatalogueSelect,
-  groupAndSortCatalogueOptions,
-} from 'components/admin/forms/zgw';
-import {get} from 'utils/fetch';
-
-// Data fetching
-
-const CATALOGUES_ENDPOINT = '/api/v2/registration/plugins/zgw-api/catalogues';
-
-const getCatalogues = async apiGroupID => {
-  const response = await get(CATALOGUES_ENDPOINT, {zgw_api_group: apiGroupID});
-  if (!response.ok) {
-    throw new Error('Loading available catalogues failed');
-  }
-  return groupAndSortCatalogueOptions(response.data);
-};
-
-// Components
-
-const CatalogueSelect = () => {
-  const {values} = useFormikContext();
-  const {zgwApiGroup = null} = values;
-
-  // fetch available catalogues
-  const {
-    loading: loadingCatalogues,
-    value: catalogueOptionGroups = [],
-    error: cataloguesError,
-  } = useAsync(async () => {
-    if (!zgwApiGroup) return [];
-    return await getCatalogues(zgwApiGroup);
-  }, [zgwApiGroup]);
-  if (cataloguesError) throw cataloguesError;
+import {CatalogueSelect as GenericCatalogueSelect} from 'components/admin/forms/zgw';
 
+const CatalogueSelect = ({loading, optionGroups}) => {
```

Comment:
This component is missing the help text

Reply:
there is none for the objects API registration either: https://open-formulieren.github.io/open-forms/?path=/story/form-design-registration-objects-api--default

I can't think of a help text that's useful that doesn't just repeat the form label. Something like "Select the catalogue" is stating the obvious, so IMO it's better to not specify one.

Reply:
Yes, it's obvious indeed, the only reason I mentioned that was because of possible connected matters like accessibility